docs/editors/settings.md

```diff
@@ -11,10 +11,42 @@ as per the editor.
 
 ### `configuration`
 
-Path to a `ruff.toml` or `pyproject.toml` file to use for configuration.
+The `configuration` setting allows you to configure Ruff's behavior directly from an editor. This
+can be done in one of the following ways:
 
-By default, Ruff will discover configuration for each project from the filesystem, mirroring the
-behavior of the Ruff CLI.
+1. **Configuration file path:** Specify the path to a `ruff.toml` or `pyproject.toml` file that
+    contains the configuration. User home directory and environment variables will be expanded.
+1. **Inline JSON configuration:** Directly provide the configuration as a JSON object. This is
+    similar to how the configuration is specified in the settings tab on the
+    [playground](https://play.ruff.sh/).
+
+!!! note "Added in Ruff `0.9.8`"
+
+    The **Inline JSON configuration** option was introduced in Ruff `0.9.8`, allowing you to define
+    settings directly in your editor without relying on an external file.
+
+By default, if no `configuration` is specified, Ruff will automatically detect and load settings
+from a configuration file in the project's filesystem, consistent with the behavior of the Ruff CLI.
```

Comment:
I find this explanation a bit confusing for two reasons:

* You say **Ruff** will automatically detect.. consistent with the behavior of the *Ruff CLI*. What I find confusing about it is that it suggests that the editor integration is the "core" Ruff and the Ruff CLI is some form of an extension
* It's unclear to me what the project's filesystem is. 

The second item seems an issue inherent to the naming of our options. Naming the option `projectFirst` might have been more intuitive

Maybe;

The default behavior, if `configuration` is unset, is to load the settings from the project's configuration (a `ruff.toml` or `pyproject.toml` in the project's directory), consistent with when running Ruff on the CLI.

Reply:
I've incorporated this suggestion, thanks